packages/testing-helpers/README.md

```diff
@@ -102,6 +102,16 @@ const el = await fixture(html`<${tag} .bar=${'baz'}></${tag}>`);
 expect(el.bar).to.equal('baz');
 ```
 
+## Customize the fixture container
+
+If you need to specify the type of element that contains your custom element fixture, (e.g., to give it an absolute position), you can pass the wrapping node in the `parentNode` option:
+
+```js
+const parentNode = document.createElement('div');
+parentNode.setAttribute('style', 'position:absolute;');
+const el = await fixture(html`<my-el></my-el>`, { parentNode });
+```
```

Comment:
maybe give an example on what the actually rendered html will be compared to the one without the custom wrapper?